src/components/ApplePayButton.tsx

```diff
@@ -0,0 +1,113 @@
+import { type StyleProp, type ViewStyle } from 'react-native';
+import { useCallback } from 'react';
+import { PlatformPayView } from '..';
+
+export const ApplePayButtonTheme = {
+  WHITE: 1,
+  WHITE_OUTLINE: 2,
+  AUTOMATIC: 3,
+  BLACK: 4,
+};
```

Comment:
```suggestion
} as const;
```

This is helpful for typescript so that this const cannot be changed elsewhere

Reply:
Addressed in [2045d8e](https://github.com/Adyen/adyen-react-native/pull/701/commits/2045d8e83db5674ef9260ef3965040e00162ddb1)